app/domain/factories/create_from_policy_factory.rb

```diff
@@ -0,0 +1,197 @@
+# frozen_string_literal: true
+
+require 'rest_client'
+require 'json_schemer'
+require 'factories/renderer'
+
+module Factories
+  class Utilities
+    def self.filter_input(str)
+      str.gsub(/[^0-9a-z\-_]/i, '')
+    end
+  end
+  class CreateFromPolicyFactory
+    def initialize(renderer: Factories::Renderer.new, http: RestClient, schema_validator: JSONSchemer, utilities: Factories::Utilities)
+      @renderer = renderer
+      @http = http
+      @schema_validator = schema_validator
+      @utilities = utilities
+
+      # JSON and URI are defined here for visibility. They are not currently
+      # mocked in testing, thus, we're not setting them in the initializer.
+      @json = JSON
+      @uri = URI
+
+      # Defined here for visibility. We shouldn't need to mock these.
+      @success = ::SuccessResponse
+      @failure = ::FailureResponse
+    end
+
+    def call(factory_template:, request_body:, account:, authorization:)
```

Comment:
This is the only file where I feel the use of `.bind` handling the `[Success|Failure]Response`s has hindered readability & upped complexity.

Reply:
This is my least favorite file in the commit. I'll take another look. I'm planning to refactor this in the future, and if it's gnarly, I'll file a ticket and give a comment here.